libraries/translations-all-v3.5.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Should this be added to the list of libraries?

Reply:
Prob not

Reply:
Why not?

Reply:
It's part of scratchblocks, just 2 files